templates/channels/channel_configuration.haml

```diff
@@ -274,7 +274,7 @@
 
     %p
       -blocktrans
-        The below endpoint should receive message this channel.
+        The below endpoint will receive messages this channel.
```

Comment:
Correct English

"The endpoint which will receives Twilio API requests for this channel"